packages/php-table-backend-utils/src/Column/Bigquery/RESTtoSQLDatatypeConverter.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\TableBackendUtils\Column\Bigquery;
+
+use Exception;
+use Keboola\Datatype\Definition\Bigquery;
+
+/**
+ * @phpstan-import-type BigqueryTableFieldSchema from Bigquery
+ */
+final class RESTtoSQLDatatypeConverter
```

Comment:
Celů logiku jak sa dělá z json SQL string sem vycucl zvlášť do třídy, protože je to komplikovanější jak je to rekurzivní.